helm_chart/ibm_storage_enabler_for_containers/templates/post-install-job.yml

```diff
@@ -0,0 +1,55 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: post-install-job
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
+  namespace: ubiquity
+  labels:
+    app: {{ template "ibm_storage_enabler_for_containers.name" . }}
+    chart: {{ template "ibm_storage_enabler_for_containers.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  template:
+    spec:
+      containers:
+      - name: hook
+        image: byrnedo/alpine-curl
+        env:
+        - name: KUBERNETES_DNS_RECORD
+          value: "kubernetes.default"
+        - name: TOKEN
+          value: "cat /var/run/secrets/kubernetes.io/serviceaccount/token"
+        - name: SERVICE_NAME
+          value: "ubiquity"
```

Comment:
don't hardcode the service name

Reply:
@27149chen  Do you mean that Service_Name should get from values.yaml?

Reply:
yes